### ISSUE TITLE
Coordinate Translation

### ISSUE THREAD

**Issue by Alexithemia** [vasturiano/3d-force-graph#268]
**Is your feature request related to a problem? Please describe.**
Same with the issue you quickly resolved on the 2D version. You said you could only do it one way which is just fine with me. Just any sort of graph x, y to screen x, y.

Also if you have any way to add events for panning, zooming, or graph rotating. Basically any sort of camera movement in the graph, with an on start and on end, so I can hook in functions when those occur would cover all my needs.

Thanks for your time.

**Comment by vasturiano** [comment]
@Alexithemia thanks for the request! I've just added the new method `graph2ScreenCoords` to this module. 
https://github.com/vasturiano/3d-force-graph#utility

As you mentioned, the reverse function maps to more than a single point and is therefore not implemented.
Available in the latest version: `1.58.0`.

As for the motion events, if you're using the default trackball controls, you can attach event handlers directly to the controller. You have `change`, `start` and `end` at your disposal:
https://threejs.org/docs/#examples/en/controls/TrackballControls

You just need to get a reference to the controller, and then attach a handler, something like:
```
myGraph.controller().addEventListener('change', event => ...)
```

**Comment by Alexithemia** [comment]
Thanks! We are using Orbit controls, as they seem to be more user friendly in our opinion and we can actually modify the camera correctly to reset it or move it. The threejs docs don't show those events for orbit, but after looking into the orbit controller code, it does appear to have the events change start and end there as well.
